syncr_backend/constants.py

```diff
@@ -34,3 +34,15 @@
 DEFAULT_DROP_METADATA_LOCATION = os.path.join(DEFAULT_INIT_DIR, "drop")
 
 DEFAULT_IGNORE = [DEFAULT_INIT_DIR]
+
+# Request types
+REQUEST_TYPE_DROP_METADATA = 1
```

Comment:
We can probably use `Enum` for these; but we can fix that later (please open an issue to fix this):
https://docs.python.org/3/library/enum.html